jenkins/envs/daint_clang_nvcc.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+source $(dirname "$BASH_SOURCE")/daint.sh
+
+module load PrgEnv-gnu
+module swap gcc/7.3.0
+
+module use /project/csstaff/fthaler/install/daint/spack/share/spack/modules/cray-cnl6-haswell
+module load llvm-9.0.0-gcc-7.3.0-3lmdwud
```

Comment:
```suggestion
module load /project/csstaff/fthaler/install/daint/spack/share/spack/modules/cray-cnl6-haswell/llvm-9.0.0-gcc-7.3.0-3lmdwud
```
and remove module use